src/main/java/org/jboss/pnc/bifrost/endpoint/LogUpload.java

```diff
@@ -93,6 +102,31 @@ public String uploadFinalLog(@Valid FinalLogUpload logUpload, @Context HttpHeade
         return "ok";
     }
 
+    @Path("/{processContext}/delete")
+    @DELETE
+    @RolesAllowed("**") // FIXME change to specific allowed roles
+    public void deleteFinalLog(
+            @PathParam("processContext") String processContext,
+            // IF tag is null, then all logs are deleted
+            @QueryParam("tag") @NotBlank String tag) {
```

Comment:
Since this endpoint is used as part of the process to delete a temporary build, I believe we don't need to specify a tag. Just delete all the final logs for that processContext.

Reply:
yep yep